content/python/concepts/modules/terms/pytorch/pytorch.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'Pytorch'
+Description: 'An open-source Python module that offers an optimized tensor library for deep learning.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Libraries'
+  - 'Modules'
+  - 'Deep Learning'
+  - 'AI'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/build-deep-learning-models-with-tensorflow'
+---
+
+**PyTorch** is a free [open-source](https://www.codecademy.com/resources/docs/general/open-source) Python module that offers an optimized tensor library for deep learning. It is based on the Torch library and can be used for applications such as natural language processing and computer vision. Released in 2016 and developed primarily by Meta (formerly Facebook) AI, deep learning software built on PyTorch include Tesla Autopilot and Uber's Pyro.
```

Comment:
Read that it's a framework -- wanting to confirm

Reply:
That seems correct. I edited the entry to reflect that.